platform/linux-generic/odp_schedule_basic.c

```diff
@@ -846,19 +846,19 @@ static inline int do_schedule_grp(odp_queue_t *out_queue, odp_event_t out_ev[],
 				 * priorities. Stop scheduling queue when pktio
 				 * has been stopped. */
 				if (pktin) {
-					int atomic = queue_is_atomic(qi);
-					int num_pkt = poll_pktin(qi, atomic);
+					int stash = !ordered;
+					int num_pkt = poll_pktin(qi, stash);
 
 					if (odp_unlikely(num_pkt < 0))
 						continue;
 
-					if (num_pkt == 0 || !atomic) {
+					if (num_pkt == 0 || !stash) {
```

Comment:
it can be if (num_pkt == 0 || ordered), stash variable looks like not needed in this patch.